Client/Frontend/Browser/Search/SearchLoader.swift

```diff
@@ -49,52 +52,59 @@ class _SearchLoader<UnusedA, UnusedB>: Loader<Cursor<Site>, SearchViewController
             }
 
             if query.isEmpty {
-                load(Cursor(status: .success, msg: "Empty query"))
+                load([Site]())
```

Comment:
Does this require `[Site]()`? Can it not be inferred based on the argument and just use `[]`